### DISCUSSION TITLE
Remap hjkl to haei

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [andmarti1424/sc-im#894] [Q&A]
I use alternative keyboard layout called [graphite-angle-kp](https://rentry.co/6bhtizdd/raw). Most of my programs mapped to use haei instead of hjkl. And I want to remap them. How can I do that?

**Comment by andmarti1424** [comment]
I would probably map a to j in normal mode, e to k and i to l. Check doc examples. Those will serve you well.

**Reply by azizsayfiddin** [reply]
Could you provide a link to that?

**Reply by andmarti1424** [reply]
Just install the app and type ` :h <enter> `
You can also find the doc file inside /src folder
